src/nupic/algorithms/KNNClassifier.py

```diff
@@ -843,33 +867,33 @@ def getPartitionId(self, i):
       return partitionId
 
 
-  def getPartitionIdPerPattern(self):
+  def getPartitionIds(self):
```

Comment:
Since it is essentially just a getter, it might be nice to match the attribute name (`getPartitionIdList`) but up to you.